src/code.rs

```diff
@@ -780,20 +831,48 @@ pub fn generate_for_table(table: &ParsedTableMacro, config: &GenerationConfig) -
     if create_struct.has_code() {
         ret_buffer.push('\n');
         ret_buffer.push_str(create_struct.code());
+        if config.options.default_impl {
+            ret_buffer.push('\n');
+            ret_buffer.push_str(
+                build_default_impl_fn(
+                    &format!("Create{struct_name}"),
+                    &create_struct.table.columns,
+                )
+                .as_str(),
+            );
+        }
+        ret_buffer.push('\n');
     }
 
     let update_struct = Struct::new(StructType::Update, table, config);
 
     if update_struct.has_code() {
         ret_buffer.push('\n');
         ret_buffer.push_str(update_struct.code());
+        if config.options.default_impl {
+            ret_buffer.push('\n');
+            ret_buffer.push_str(
+                build_default_impl_fn(
+                    &format!("Update{struct_name}"),
+                    &update_struct.table.columns,
+                )
+                .as_str(),
+            );
+        }
+        ret_buffer.push('\n');
```

Comment:
`Default` is already derived for `Update` structs

https://github.com/Wulf/dsync/blob/2797966da4ab58f5a86df559fc29452cda15be75/test/simple_table_pg/models/todos/generated.rs#L48